codersdk/name_test.go

```diff
@@ -254,3 +256,40 @@ func TestUserRealNameValid(t *testing.T) {
 		})
 	}
 }
+
+func TestGroupNameValid(t *testing.T) {
+	t.Parallel()
+
+	testCases := []struct {
+		Name  string
+		Valid bool
+	}{
+		{"", false},
+		{"my-group", true},
+		{"create", false},
+		{"new", false},
+		{"Lord Voldemort Team", false},
+		{randomString(255), true},
+		{randomString(256), false},
```

Comment:
Could you use `testutil.GetRandomName(...)` ?

Reply:
I think get random name is limited in length 🤔

Reply:
oh, then perhaps `namesgenerator.GetRandomName(255)`?

Reply:
`GetRandomName` only checks if the argument is greater than 0, and if it is, appends a random number between 1 and 10

Reply:
ok, one last guess: `cryptorand.String(...)`